testkit-backend/streamlogger.go

```diff
@@ -39,3 +39,6 @@ func (l *streamLog) Infof(name string, id string, msg string, args ...interface{
 func (l *streamLog) Debugf(name string, id string, msg string, args ...interface{}) {
 	l.writeLine(fmt.Sprintf("[%s %s] %s", name, id, fmt.Sprintf(msg, args...)))
 }
+func (l *streamLog) Tracef(name string, id string, msg string, args ...interface{}) {
```

Comment:
Need empty line

Reply:
I'll add empty lines for the other functions in the file as well in that case.